Automattic/jetpack
# Hide stats dashboard widget for users below view_Stats cap

## Proposed changes:

* Removes the `wp-admin` dashboard Jetpack Stats widget for users lower than the `edit_posts` capability.
* Adds a filter to allow for this to be changed programatically.

Team51 noticed this issue whilst working with a partner who's subscribers have access to the `wp-admin` profile pages and dashboard. These stats seem like they should be hidden for certain user roles. `edit_posts` is a suggestion of the capability level I think would warrant access to this information

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/23197

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Go to your wp-admin dashboard.
* If you're logged in as a user with `edit_posts` capability or higher, ensure the stats widget can be seen.
* If you're logged in as a user without `edit_posts` capability, ensure the stats widget cannot be seen.
